releasenotes/notes/29930.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: release-notes/v2
+kind: bug-fix
+area: istioctl
+issue:
+  - 29930
+
+releaseNotes:
+- |
+  **Fixed** `istioctl install` read empty istiooperator from standard input.
```

Comment:
I'm not sure we need a release note if the fix is for something that wasn't shipped. At least I think what is fixed is correct in the 1.8 branch, and this was a regression from 1.8, and never shipped. That said, I'm fine leaving this in.

Reply:
Thanks @ericvn
removed the release note.